src/hotspot/share/opto/phasetype.hpp

```diff
@@ -186,13 +195,10 @@ class PhaseNameValidator {
     }
   }
 
-  bool is_valid() const {
-    return _valid;
-  }
-
-  const char* what() const {
-    return _bad;
-  }
+  const BitMap& mask() const { return _mask; }
```

Comment:
You could additionally assert `is_valid` before returning the mask, just to ensure it is never called when invalid.

Reply:
Thanks, added now.